ar4_mujoco_sim/package.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<package format="3">
+  <name>ar4_mujoco_sim</name>
+  <version>0.0.1</version>
+  <description>Launch Mujoco simulation of the AR4</description>
+  <author email="glpuga@ekumenlabs.com">Gerardo Puga</author>
+  <maintainer email="glpuga@ekumenlabs.com">Gerardo Puga</maintainer>
+  <license file="LICENSE">BSD-3-Clause</license>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
+  <exec_depend>ar4_moveit_config</exec_depend>
+  <exec_depend>controller_manager</exec_depend>
+  <exec_depend>mujoco_ros2_control</exec_depend>
```

Comment:
I got some issues with this package when doing `colcon build` in `ar4_gazebo`, is this expected?

```
--- stderr: mujoco_ros2_control                                                                                     
/home/developer/ws/src/external/mujoco_ros2_control/mujoco_ros2_control/src/mujoco_rendering.cpp: In member function ‘void mujoco_ros2_control::MujocoRendering::keyboard_callback_impl(GLFWwindow*, int, int, int, int)’:
/home/developer/ws/src/external/mujoco_ros2_control/mujoco_ros2_control/src/mujoco_rendering.cpp:113:58: warning: unused parameter ‘window’ [-Wunused-parameter]
  113 | void MujocoRendering::keyboard_callback_impl(GLFWwindow* window, int key, int scancode, int act, int mods)
```

Reply:
If you just do `colcon build` from the isaac docker you'll get plenty of errors because Mujoco is not installed and therefore `mujoco_ros2_control` cannot be buillt.